### PR TITLE
add ticket button

### PR DESCRIPTION
**Pull request by VictoriaXiao**

### CODE REVIEW

File: _layouts/singer.html [hunk 1]
```diff
@@ -20,4 +20,52 @@ <h2>2022歌手赛：</h2>
...
+            widgetType: 'checkout',
+            eventId: '355595625087',
+            iframeContainerId: 'eventbrite-widget-container',
+
+            // Optional
+            iframeContainerHeight: iframeContainerHeight,  // Widget height in pixels. Defaults to a minimum of 425px if not provided
+            onOrderComplete: exampleCallback  // Method called when an order has successfully completed
+        });
+    </script>
+</div>
```
Comment (zzhai) [inline _layouts/singer.html]: Line 69. where is the opening _div_ ?

Comment (zzhai) [inline _layouts/singer.html]:
```suggestion
```

Comment (VictoriaXiao) [reply _layouts/singer.html]: <div class="container">

Comment (VictoriaXiao) [reply _layouts/singer.html]: > opening
<div class="container">

Comment (VictoriaXiao) [reply _layouts/singer.html]: <div class="container">

**Review by zzhai — APPROVED** [review]

**Pull request merged**